pkg/cmd/admin/prune/images.go

```diff
@@ -88,6 +89,7 @@ func NewCmdPruneImages(f *clientcmd.Factory, parentName, name string, out io.Wri
 	cmd.Flags().IntVar(&opts.KeepTagRevisions, "keep-tag-revisions", opts.KeepTagRevisions, "Specify the number of image revisions for a tag in an image stream that will be preserved.")
 	cmd.Flags().StringVar(&opts.CABundle, "certificate-authority", opts.CABundle, "The path to a certificate authority bundle to use when communicating with the managed Docker registries. Defaults to the certificate authority data from the current user's config file.")
 	cmd.Flags().StringVar(&opts.RegistryUrlOverride, "registry-url", opts.RegistryUrlOverride, "The address to use when contacting the registry, instead of using the default value. This is useful if you can't resolve or reach the registry (e.g.; the default is a cluster-internal URL) but you do have an alternative route that works.")
+	cmd.Flags().BoolVar(&opts.ExceedLimits, "exceed-limits", opts.ExceedLimits, "Specify that images which are exceeding limits (see 'openshift.io/Image') should be considered a candidate for pruning.")
```

Comment:
`--keep-limits-exceeded` sounds like it's telling you to exceed limits.
Use more words:  `--keep-over-size-limit-images`

On Thu, Jun 23, 2016 at 2:31 PM, Jessica Forrester <notifications@github.com
> wrote:

> In pkg/cmd/admin/prune/images.go
> <https://github.com/openshift/origin/pull/9481#discussion_r68289708>:
>
> > @@ -88,6 +89,7 @@ func NewCmdPruneImages(f *clientcmd.Factory, parentName, name string, out io.Wri
> >  	cmd.Flags().IntVar(&opts.KeepTagRevisions, "keep-tag-revisions", opts.KeepTagRevisions, "Specify the number of image revisions for a tag in an image stream that will be preserved.")
> >  	cmd.Flags().StringVar(&opts.CABundle, "certificate-authority", opts.CABundle, "The path to a certificate authority bundle to use when communicating with the managed Docker registries. Defaults to the certificate authority data from the current user's config file.")
> >  	cmd.Flags().StringVar(&opts.RegistryUrlOverride, "registry-url", opts.RegistryUrlOverride, "The address to use when contacting the registry, instead of using the default value. This is useful if you can't resolve or reach the registry (e.g.; the default is a cluster-internal URL) but you do have an alternative route that works.")
> > +	cmd.Flags().BoolVar(&opts.ExceedLimits, "exceed-limits", opts.ExceedLimits, "Specify that images which are exceeding limits (see 'openshift.io/Image') should be considered a candidate for pruning.")
>
> personally i think --keep-exceeded-limits reads wrong
> … <#m_-7427032287492723090_>
> --keep-limits-exceeded or --keep-exceeds-limits
> On Thu, Jun 23, 2016 at 10:30 AM, Clayton Coleman ***@***.***> wrote: In
> pkg/cmd/admin/prune/images.go <#9481 (comment)
> <https://github.com/openshift/origin/pull/9481#discussion_r68244946>>: >
> @@ -88,6 +89,7 @@ func NewCmdPruneImages(f *clientcmd.Factory, parentName,
> name string, out io.Wri > cmd.Flags().IntVar(&opts.KeepTagRevisions,
> "keep-tag-revisions", opts.KeepTagRevisions, "Specify the number of image
> revisions for a tag in an image stream that will be preserved.") >
> cmd.Flags().StringVar(&opts.CABundle, "certificate-authority",
> opts.CABundle, "The path to a certificate authority bundle to use when
> communicating with the managed Docker registries. Defaults to the
> certificate authority data from the current user's config file.") >
> cmd.Flags().StringVar(&opts.RegistryUrlOverride, "registry-url",
> opts.RegistryUrlOverride, "The address to use when contacting the registry,
> instead of using the default value. This is useful if you can't resolve or
> reach the registry (e.g.; the default is a cluster-internal URL) but you do
> have an alternative route that works.") > +
> cmd.Flags().BoolVar(&opts.ExceedLimits, "exceed-limits", opts.ExceedLimits,
> "Specify that images which are exceeding limits (see 'openshift.io/Image')
> should be considered a candidate for pruning.") Use --keep-exceeded-limits
> and default it to true. — You are receiving this because you are on a team
> that was mentioned. Reply to this email directly, view it on GitHub <
> https://github.com/openshift/origin/pull/9481/files/1c5d96ed500d7748adbc3ca24dab14d19df537e4..827538449d912747a01a3a85e0f5be714676ae25#r68244946>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/ABZk7YgjDwn5CNmX9cTL_PR0q83ud5GGks5qOpiXgaJpZM4I7qnF>
> .
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9481/files/1c5d96ed500d7748adbc3ca24dab14d19df537e4..827538449d912747a01a3a85e0f5be714676ae25#r68289708>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p0VtkbszSnNDrqIqI11A_wZcVvZWks5qOtDmgaJpZM4I7qnF>
> .
>